src/theme.c

```diff
@@ -750,8 +755,9 @@ create_corners(struct theme *theme)
 static void
 post_processing(struct theme *theme)
 {
-	theme->title_height = font_height(&rc.font_activewindow)
-		+ 2 * theme->padding_height;
+	int h = font_height(&rc.font_activewindow) + 2 * theme->padding_height;
+	theme->title_height = MAX(theme->title_height, h);
```

Comment:
Suggest changing this to the code below so that `theme.titlebar_height` doesn't just apply when it's greater than font-height + 2 * padding_height.

```
/* Fallback on font-height + 2x padding.height if not set */
if (theme->title_height > 0) {
	theme->title_height = font_height(&rc.font_activewindow)
		+ 2 * theme->padding_height;
}
```

Reply:
Sorry, should of course be:

```
int h = font_height(&rc.font_activewindow);
if (theme->title_height < h) {
	theme->title_height = h + 2 * theme->padding_height;
}
```

...to avoid titlebar being smaller than than font-height which would be no good.